### PR TITLE
(+) Added support to locate the arrow in a DropDownButton - without default button - via ArrowLocation property

### PR DESCRIPTION
**Pull request by alexfdezsauco**
### Checklist

- [X] I have included examples or tests
- [X] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

- (+) Added support to locate the arrow in a DropDownButton - without default button - via ArrowLocation property

### CODE REVIEW

File: src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Converters/ArrowLocationConverter.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    using System.Windows.Controls;
+    using Catel.MVVM.Converters;
+
+    public class ArrowLocationConverter : ValueConverterBase
+    {
+        protected override object Convert(object value, Type targetType, object parameter)
+        {
+            if (targetType == typeof(Dock))
+            {
+                return Enum.Parse(targetType, value.ToString(), true);
```
Comment (GeertvanHorrik) [inline src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Converters/ArrowLocationConverter.cs]: value can be null. Try to use ObjectTostringConverter or Helper and check for null.

File: src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Converters/ArrowLocationConverter.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            }
+
+            if (targetType == typeof(HorizontalAlignment))
+            {
+                if ((ArrowLocation)value == ArrowLocation.Top || (ArrowLocation)value == ArrowLocation.Bottom)
```
Comment (GeertvanHorrik) [inline src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Converters/ArrowLocationConverter.cs]: cast once & store in variable

File: src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Views/DropDownButton.xaml.cs [hunk 3]
```diff
@@ -13,8 +13,32 @@ namespace Orc.Controls
     using System.Windows.Input;
     using System.Windows.Media;
     using Catel;
-    using Catel.MVVM.Views;
-    using Microsoft.WindowsAPICodePack.Shell;
+
+    /// <summary>
+    /// The arrow location.
+    /// </summary>
+    public enum ArrowLocation
```
Comment (GeertvanHorrik) [inline src/Orc.Controls/Orc.Controls.Shared/Controls/DropDownButton/Views/DropDownButton.xaml.cs]: Maybe name it "DropDownArrowLocation"? We might need this in other controls as well in the future.

**Comment by alexfdezsauco** [comment]
Fixed

**Pull request merged**